VideoLocker/res/layout/fragment_course_unit_video.xml

```diff
@@ -28,47 +28,46 @@
     </LinearLayout>
 
     <LinearLayout
+        android:id="@+id/player_container"
         android:layout_width="match_parent"
         android:layout_height="200dp"
-        android:orientation="vertical"
-        android:id="@+id/player_container">
+        android:orientation="vertical">
 
     </LinearLayout>
 
     <LinearLayout
```

Comment:
Are all these numbers going to be right on a differently sized device? Should we be centering the icon+text vertically in whatever area it has? (Or is that what's happening and I'm just bad at reading layout xml?)